build/root/Makefile

```diff
@@ -584,6 +585,9 @@ define BAZEL_TEST_INTEGRATION_HELP_INFO
 endef
 bazel-test-integration:
 	@echo "$$BAZEL_TEST_INTEGRATION_HELP_INFO"
+else ifeq ($(UNAME_M),aarch64)
+bazel-test-integration:
+	bazel test --action_env=ETCD_UNSUPPORTED_ARCH=arm64 --config integration //test/integration/...
```

Comment:
would it be better to set this environment variable in the test framework itself, i.e. in `test/integration/framework/etcd.go`?

also, is this environment variable documented somewhere?

Reply:
Done.
Please see the document: https://github.com/etcd-io/etcd/blob/a621d807f061e1dd635033a8d6bc261461429e27/Documentation/op-guide/supported-platform.md